### ISSUE TITLE
Remix failed to fetch

### ISSUE THREAD

**Issue by WilliamDjo** [ethereum/remix-project#3763]
Why can't I push my code to github from the remix IDE? Whenever I try to push, it says "Error from IDE: failed to fetch". Same thing happens as well when I try to pull.
<img width="409" alt="Screenshot 2023-06-06 at 2 46 10 pm" src="https://github.com/ethereum/remix-project/assets/66203375/77a55d7e-d5c0-4418-9937-5b2306dc0980">

**Comment by WilliamDjo** [comment]
> Hi, do you have your github token setup? And is it still valid? For private repositories you need a token that has access to write. Can you verify you have that setup?

Yes I do have it setup and it is valid. Btw the repository I made is public. 
![Screenshot 2023-06-08 at 2 11 20 pm](https://github.com/ethereum/remix-project/assets/66203375/db98fc22-f51d-4284-ba16-eb259f97bcd9)

**Comment by bunsenstraat** [comment]
So are you able to clone it in remix but not push or pull or fetch?

**Comment by bunsenstraat** [comment]
Often the JavaScript console might provide more info on the error too.

**Comment by WilliamDjo** [comment]
> So are you able to clone it in remix but not push or pull or fetch?

Yes

**Comment by bunsenstraat** [comment]
I have cloned https://github.com/WilliamDjo/COMP6452 and fetched, it worked. Didn't try to push yet. 
Is the remote listed as origin in the remotes section?

**Comment by WilliamDjo** [comment]
> I have cloned https://github.com/WilliamDjo/COMP6452 and fetched, it worked. Didn't try to push yet. Is the remote listed as origin in the remotes section?

It's listed as main

**Comment by WilliamDjo** [comment]
![Screenshot 2023-06-09 at 2 02 49 am](https://github.com/ethereum/remix-project/assets/66203375/bbcfa326-fe42-4431-bf9e-c754afc4e583)

**Comment by bunsenstraat** [comment]
can you try adding the remote as 'origin' and not as williamdjo ... there seems to be a bug.

**Comment by bunsenstraat** [comment]
can you check the console and see what it says when you fetch or pull. 
![Screenshot 2023-06-09 at 08 34 05](https://github.com/ethereum/remix-project/assets/2787300/b6f7c7d4-a7c5-413b-9073-5e9dc1d7e97c)

**Comment by WilliamDjo** [comment]
> can you check the console and see what it says when you fetch or pull. ![Screenshot 2023-06-09 at 08 34 05](https://user-images.githubusercontent.com/2787300/244611428-b6f7c7d4-a7c5-413b-9073-5e9dc1d7e97c.png)
![Screenshot 2023-06-09 at 5 01 39 pm](https://github.com/ethereum/remix-project/assets/66203375/78eb7012-ce6f-434f-9f97-fc353d901616)

**Comment by WilliamDjo** [comment]
> can you check the console and see what it says when you fetch or pull. ![Screenshot 2023-06-09 at 08 34 05](https://user-images.githubusercontent.com/2787300/244611428-b6f7c7d4-a7c5-413b-9073-5e9dc1d7e97c.png)

Do you know whats wrong? I still am getting the same error :(

**Comment by bunsenstraat** [comment]
It's all very weird... I will think about it.

**Comment by bunsenstraat** [comment]
I deployed a fix for the FETCH command. Can you try again please? It was a bug!
